cookbooks/aws-parallelcluster-slurm/templates/default/slurm/head_node/update_munge_key.sh.erb

```diff
@@ -13,7 +13,7 @@ SECRET_ARN="<%= @munge_key_secret_arn %>"
 REGION="<%= @region %>"
 MUNGE_USER="<%= @munge_user %>"
 MUNGE_GROUP="<%= @munge_group %>"
-CLUSTER_USER="<%= @cluster_user %>"
+SHARED_DIRECTORY="<%= @shared_directory %>"
```

Comment:
Done!

Reply:
If you have 5 minutes, please rename `SHARED_DIRECTORY` to `SHARED_DIRECTORY_COMPUTE` so that it's immediately distinguishable from `SHARED_DIRECTORY_LOGIN`.